src/libNode/FinalBlockProcessing.cpp

```diff
@@ -913,7 +910,7 @@ bool Node::ProcessFinalBlockCore(const bytes& message, unsigned int offset,
     if (toSendPendingTxn) {
       SendPendingTxnToLookup();
     }
-
+    ClearUnconfirmedTxn();
```

Comment:
Are we missing clearing `UnconfirmedTxn` and `ClearPendingAndDroppedTxn` for lookup nodes `LOOKUP_NODE_MODE = true`.